.nvmrc

```diff
@@ -1 +1 @@
-16.19.0
+20.11.0
```

Comment:
I chose 20.11.0 as that is the latest LTS by Node.js https://nodejs.org/en/blog/release/